core/lib/libtomcrypt/sub.mk

```diff
@@ -114,6 +114,7 @@ _CFG_CRYPTO_WITH_ASN1 := $(call cryp-one-enabled, RSA DSA ECC)
 _CFG_CRYPTO_WITH_FORTUNA_PRNG := $(call cryp-all-enabled, AES SHA256)
 
 cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -DLTC_SMALL_CODE -DLTC_NO_FAST
+cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -Os
```

Comment:
How is it not the case? I do see `-Os` being used when LTC is built, for instance:

>  $ make clean && make out/arm-plat-vexpress/core/lib/libtomcrypt/src/pk/rsa/rsa_encrypt_key.o V=1 | grep rsa_encrypt_key.o | grep -- -Os
>  CLEAN   .
> arm-linux-gnueabihf-gcc -std=gnu99 -Werror -fdiagnostics-show-option -Wall -Wcast-align -Werror-implicit-function-declaration -Wextra -Wfloat-equal -Wformat-nonliteral -Wformat-security -Wformat=2 -Winit-self -Wmissing-declarations -Wmissing-format-attribute -Wmissing-include-dirs -Wmissing-noreturn -Wmissing-prototypes -Wnested-externs -Wpointer-arith -Wshadow -Wstrict-prototypes -Wswitch-default -Wwrite-strings -Wno-missing-field-initializers -Wno-format-zero-length -Waggregate-return -Wredundant-decls -Wold-style-definition -Wstrict-aliasing=2 -Wundef -pedantic -Wdeclaration-after-statement -Os -g -ffunction-sections -fdata-sections -pipe -g3 -mcpu=cortex-a15 -mno-apcs-float -mfloat-abi=soft -mthumb -mthumb-interwork -fno-short-enums -fno-common -mno-unaligned-access -MD -MF out/arm-plat-vexpress/core/lib/libtomcrypt/src/pk/rsa/.rsa_encrypt_key.o.d -MT out/arm-plat-vexpress/core/lib/libtomcrypt/src/pk/rsa/rsa_encrypt_key.o -nostdinc -isystem /usr/lib/gcc-cross/arm-linux-gnueabihf/5/include -Ilib/libutils/isoc/include -Ilib/libutils/ext/include -Ilib/libmpa/include -Icore/lib/libtomcrypt/include -Icore/arch/arm/plat-vexpress/. -D__KERNEL__ -DPLATFORM_FLAVOR=PLATFORM_FLAVOR_ID_qemu_virt -Icore/include -include out/arm-plat-vexpress/include/generated/conf.h -Iout/arm-plat-vexpress/core/include/generated -Icore/arch/arm/include -DARM32=1 -D__ILP32__=1 -DTRACE_LEVEL=1 -Ilib/libutee/include -DLTC_SMALL_CODE -DLTC_NO_FAST -c core/lib/libtomcrypt/src/pk/rsa/rsa_encrypt_key.c -o out/arm-plat-vexpress/core/lib/libtomcrypt/src/pk/rsa/rsa_encrypt_key.o

`-Os` is there as expected due to `platform-cflags-optimization ?=  -Os` in `core/arch/arm/arm.mk`. Am I missing something?

So I see no need to do anything regarding `-Os`. However it may be valid to keep CFG_CRYPTO_SIZE_OPTIMIZATION to enable `-DLTC_SMALL_CODE`, I don't know.